lib/project.rb

```diff
@@ -170,6 +173,7 @@ def init_workspace
         @logger.warn('Workspace path already exists')
       end
       @logger.info("Workspace path: #{@workspace_path}")
+      @setup_manager_type&.enter @workspace_path
```

Comment:
Why we don't need to enter workspace_path if it is specified? 

I am thinking about the next task: `Add ability to bring up previous session (for example to get dump from)` 

I expect that  auto_hck will load/initialize the state of every component when  workspace_path is provided.

Maybe we should have 2 different options for this one for internal use and the other for public usage?

Reply:
> Maybe we should have 2 different options for this one for internal use and the other for public usage?

Perhaps so, but it is out of scope of this change.